lib/chef/knife/bootstrap/common_bootstrap_options.rb

```diff
@@ -1,7 +1,7 @@
 #
-# Author:: Aliasgar Batterywala (aliasgar.batterywala@clogeny.com)
+# Author:: Nimesh Patni (nimesh.patni@msystechnologies.com)
```

Comment:
Unless a file is from scratch, we should leave the old author as another line. It doesn't hurt and it gives thanks to past developers.

Reply:
Right, apologies. Actually I was attempting to refactor the changes from scratch, changed my mind later. Forgot to remove this change before commit. Thanks for notice.